tests/helpers/studio/test_utils.py

```diff
@@ -1,695 +1,495 @@
 import os
```

Comment:
Review note: The old version was generated by ai and not active (marked with pytest.skip). The new version I also generated a first pass with AI but completely reworked it. It does not completely test all cases, but the coverage is good enough for now imho.